noark-extraction-validator/src/main/java/com/documaster/validator/validation/collector/ValidationCollector.java

```diff
@@ -85,209 +77,68 @@ public int getTotalErrorCount() {
 		return totalErrorCount;
 	}
 
-	public int getInformationCountIn(String group) {
+	public int getInformationCountIn(String groupName) {
+
+		return informationCount.containsKey(groupName) ? informationCount.get(groupName) : 0;
+	}
+
+	public int getWarningCountIn(String groupName) {
+
+		return warningCount.containsKey(groupName) ? warningCount.get(groupName) : 0;
+	}
+
+	public int getErrorCountIn(String groupName) {
 
-		return informationCount.containsKey(group) ? informationCount.get(group) : 0;
+		return errorCount.containsKey(groupName) ? errorCount.get(groupName) : 0;
 	}
 
-	public int getWarningCountIn(String group) {
+	public int getTotalResultCountIn(String groupName) {
 
-		return warningCount.containsKey(group) ? warningCount.get(group) : 0;
+		return getInformationCountIn(groupName) + getWarningCountIn(groupName) + getErrorCountIn(groupName) + 1;
```

Comment:
The `+1` should have been in the `getNextIdForGroup()` method in order to make the IDs 1-based. Moved it to where ti belongs.